financial-templates-lib/test/clients/TokenBalanceClient.js

```diff
@@ -35,66 +35,48 @@ contract("BalanceMonitor.js", function(accounts) {
   });
 
   it("Returning token balances", async function() {
-    // Should start at empty state (null) and the address should not be resolved.
```

Comment:
`null` tests are not needed anymore. balances are populated immediately after calling the `get*` functions.